pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "linkml"
-version = "0.0.0"
+version = "1.7.5.post241.dev0+8fd9c529"
```

Comment:
We should leave this as `"0.0.0"` as indication that dynamic versioning is being used.

Reply:
Right my bad. I havent figured out what triggers it but sometimes that field automatically replaces itself

Reply:
done in https://github.com/linkml/linkml/pull/2208/commits/7fed49b3764d3f8c1ee84422444e675d0baec99c